src/components/structures/MessagePanel.tsx

```diff
@@ -347,8 +347,8 @@ export default class MessagePanel extends React.Component<IProps, IState> {
         return this.eventTiles[eventId]?.ref?.current;
     }
 
-    public getTileForEventId(eventId: string): UnwrappedEventTile {
-        if (!this.eventTiles) {
+    public getTileForEventId(eventId?: string): UnwrappedEventTile | undefined {
+        if (!this.eventTiles || eventId === undefined || eventId === null) {
```

Comment:
we can simplify this to `!eventId` - an empty string as an event ID is impractical and should be impossible (if it somehow makes it this far in the stack, the calling code deserves to explode imo :p )

Reply:
```suggestion
        if (!this.eventTiles || !eventId) {
```